django_app/redbox_app/redbox_core/views.py

```diff
@@ -127,6 +112,13 @@ def upload_view(request):
 
             try:
                 api.upload_file(uploaded_file.name, request.user)
+                file = models.File.objects.create(
```

Comment:
does this ORM code to be outside of this try/except?

Reply:
I think it will stay inside as the except just adds errors. If I move it after+outside the try/except then if the upload fails django will still save the file. If the upload fails then the saving should fail too